modules/apps/object/object-test/src/testFunctional/macros/ObjectNotifications.macro

```diff
@@ -80,17 +80,12 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
-	macro deleteQueue {
-		if (IsElementPresent(key_statusName = ${status}, locator1 = "ObjectNotifications#NOTIFICATION_STATUS")) {
-			Click(
-				key_text = ${triggerBy},
-				locator1 = "Icon#ANY_VERTICAL_ELLIPSIS");
+	macro deleteNotificationEntry {
+		Click(
+			key_text = ${triggerBy},
+			locator1 = "ObjectNotifications#DELETE_ENTRY_NOTIFICATION_ON_QUEUE");
 
-			MenuItem.clickNoError(menuItem = "Delete");
-		}
-		else {
-			Click(locator1 = "Icon#MOVE_TO_RECYCLE_BIN");
-		}
+		Alert.viewSuccessMessage();
```

Comment:
In general, I don't recommend asserting the success message. It's fragile and doesn't provide a lot of value to our tests, especially since we're using this macro in a `Given` or `When` step.